src/main.js

```diff
@@ -55,34 +59,50 @@ function deleteIdea(){
     ideaCardsGrid.removeChild(target);
     for(var i = 0; i < list.length; i++){
       if (target.getAttribute("id") == list[i].id) {
-        list.splice(i , 1);
+        list[i].deleteFromStorage(target);
+        list.splice(i, 1);
       };
     };
   };
 };
 
 function toggleFavoriteCard(){
-  var favoriteIcon = event.target.src;
   var target = event.target.closest(".idea-card")
   if(event.target.src.includes('star.svg')) {
-    event.target.src = './src/icons/star-active.svg'
+    event.target.src = './src/icons/star-active.svg';
     for(var i = 0; i < list.length; i++){
       if(target.getAttribute("id") == list[i].id) {
+        var updateIdeaCard = new Idea(list[i].title, list[i].body, true);
+        updateIdeaCard.id = list[i].id;
         list[i].star = true;
+        list[i].updateIdea(updateIdeaCard);
       }
     }
   } else {
     event.target.src = './src/icons/star.svg'
     for(var i = 0; i < list.length; i++){
       if(target.getAttribute("id") == list[i].id) {
+        var updateIdeaCard = new Idea(list[i].title, list[i].body);
+        updateIdeaCard.id = list[i].id;
         list[i].star = false;
+        list[i].updateIdea(updateIdeaCard);
       }
     }
   }
 }
 
-
 function displayIdeaCards(){
+  if (localStorage.length > 0) {
+    list = [];
+    for (var i = 0; i < localStorage.length; i++) {
+      var retrievedIdeaCard = localStorage.getItem(`${Object.keys(localStorage)[i]}`);
+      var parsedIdeaCard = JSON.parse(retrievedIdeaCard);
+      var rebuildIdeaCard = new Idea(parsedIdeaCard.title, parsedIdeaCard.body, parsedIdeaCard.star);
+      rebuildIdeaCard.id = parsedIdeaCard.id;
+      console.log(rebuildIdeaCard);
```

Comment:
Well done. I like how you used localStorage to identify the card with a unique name. That will be helpful when we need to display, delete, toggle favorite certain cards without having to know the specific id of the card.

Also, I see a console.log here on 102 that we can refactor